examples/src/main/java/org/apache/ignite/example/storage/VolatilePageMemoryStorageExample.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.example.storage;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * This example demonstrates a usage of the PageMemory storage engine configured with an in-memory data region.
+ *
+ * <p>To run the example, do the following:
+ * <ol>
+ *     <li>Import the examples project into you IDE.</li>
+ *     <li>
+ *         Download and prepare artifacts for run Ignite node using the CLI tool (if not done yet):<br>
+ *         {@code ignite init}
+ *     </li>
+ *     <li>
+ *         Add module "ignite-storage-page-memory" using the CLI tool (if not done yet):<br>
+ *         {@code ignite module add mvn:org.apache.ignite:ignite-storage-page-memory:version}<br>
+ *         For example:<br>
+ *         {@code ignite module add mvn:org.apache.ignite:ignite-storage-page-memory:3.0.0-SNAPSHOT}<br>
+ *     </li>
+ *     <li>
+ *         Start a server node using the CLI tool:<br>
+ *         {@code ignite node start --config=$IGNITE_HOME/examples/config/ignite-config.json my-first-node}
+ *     </li>
+ *     <li>
+ *         Cluster initialization using the CLI tool (if not done yet):<br>
+ *         {@code ignite cluster init --cluster-name=ignite-cluster --node-endpoint=localhost:10300 --meta-storage-node=my-first-node}
+ *     </li>
+ *     <li>
+ *         Add configuration for in-memory data region of of the PageMemory storage engine using the CLI tool (if not done yet):<br>
+ *         {@code ignite config set --type=cluster "{pageMemory:{regions:[{name:in-memory,persistent:false}]}}"}
```

Comment:
Actually, I would advice to use simplified syntax. Here, it's enough to do the following:
`pageMemory.regions.in-memory: { persistent = false }` we don't need that many braces. Also, how common is it to use dashes in names? I don't know